docs/bond-rest-api-spec.yaml

```diff
@@ -0,0 +1,1303 @@
+          type: number
+          format: uint64
+        quantity:
+          description: The quantity the firm wishes to buy or sell in USD as a multiple of the bond's par value
+          type: number
+          format: uint64
+        quote_id:
+          description: Quote which is matched to this order
+          type: string
+        status:
+          description: Whether the order is settled, open, or matched
+          type: string
+          enum:
+            - OPEN
+            - SETTLED
+            - MATCHED
+        timestamp:
+          description: Time that the order was created (UTC Timestamp)
+          type: number
+          format: uint64
+    OrganizationModel:
+      description: An organization model
+      type: object
+      properties:
+        organization_id:
+          description: Ticker symbol or pricing source ID of an organization
+          type: string
+        name:
+          description: Name of an organization
+          type: string
+        organization_type:
+          $ref: '#/components/schemas/OrganizationTypeEnum'
+        industry:
+          description: Industry of operation for an organization
+        holdings:
+          type: array
+          items:
+            $ref: '#/components/schemas/HoldingModel'
+        authorizations:
+          type: array
+          items:
+            $ref: '#/components/schemas/AuthorizationModel'
+        receipts:
+          type: array
+          items:
+            $ref: '#/components/schemas/ReceiptModel'
+    ParticipantModel:
+      description: A participant model
+      properties:
+        public_key:
+          description: Participant's public key
+          type: string
+        username:
+          description: Participant's username
+          type: string
+        organization_id:
+          description: ID of the organization the participant is associated with
+          type: string
+    QuoteModel:
+      description: quote
+      properties:
+        quote_id:
+          description: Unique ID of the quote
+          type: string
+        organization_id:
+          description: Organization that the quote is on behalf of
+          type: string
+        bond_id:
+          description: ID of the bond the quote is for
+          type: string
+        ask_price:
+          description: Minimum price for which the firm is willing to sell the bond (millionths of dollars)
+          type: number
+          format: uint64
+        ask_qty:
+          description: Maximum quantity the firm is willing to sell
+          type: number
+          format: uint64
+        bid_price:
+          description: Maximum price for which the firm is willing to sell the bond (millionths of dollars)
+          type: number
+          format: uint64
+        bid_qty:
+          description: Minimum quantity the firm is willing to sell
```

Comment:
Need to alter bid_price and bid_qty description to accurately describe action (sell => buy)